distribution/zip/ballerina/bin/ballerina

```diff
@@ -199,8 +199,6 @@ fi
 $JAVACMD \
 	-Xbootclasspath/a:"$BALLERINA_XBOOTCLASSPATH" \
 	-Xms256m -Xmx1024m \
-	-XX:+HeapDumpOnOutOfMemoryError \
-	-XX:HeapDumpPath="$BALLERINA_HOME/heap-dump.hprof" \
```

Comment:
Is it a good idea to get rid of these?

Reply:
IMO, we should not have added these in the first place since this will expose the java level details to the end users. Also, these flags are needed for Ballerina language developers to debug issues in the language, so I don't think the end users want this level of details. 

If the end users want to get a heap dump, they can add these flags when they run the ballerina code.

@hasithaa / @lankavitharana / @SupunS - WDYT?

Reply:
IMO it is ok to keep those flags default in ballerina script. This is because we already have other java level details in the script. If OOM is intermittent , it might not easily reproducible.  Hence already created heap dump will be useful to analyse any OOM cases.

Reply:
@warunalakshitha - If that is the case, we will have to close this issue since there is no other way to prevent `java.lang.OutOfMemoryError: Java heap space` error message.

Reply:
For that we need to redirect the stderr stream when the ballerina runtime first initializes and capture these sort of errors in a custom stream of our own and handle in an appropriate manner.